### PR TITLE
[fr] #3186, #3184, #3182, #3178

### PR DESCRIPTION
**Pull request by cake17**
@antograssiot I'm sure I missed some of the last changes

### CODE REVIEW

File: fr/console-and-shells.rst [hunk 1]
```diff
@@ -428,6 +428,15 @@ La classe ``Shell`` fournit quelques méthodes pour afficher le contenu::
     // Ecrire avec stderr et arrêter le processus
     $this->error('Erreur Fatale');
 
+Elle fournit aussi deux méthodes pratiques en ce qui concerne le niveau de
+sortie::
+
+    // Would only appear when verbose output is enabled (-v)
```
Comment (HavokInspiration) [inline fr/console-and-shells.rst]: Comment in English

File: fr/console-and-shells.rst [hunk 2]
```diff
@@ -428,6 +428,15 @@ La classe ``Shell`` fournit quelques méthodes pour afficher le contenu::
...
+    $this->verbose('Verbose message');
+
+    // Would appear at all levels.
```
Comment (HavokInspiration) [inline fr/console-and-shells.rst]: Comment in English

File: fr/core-libraries/email.rst [hunk 3]
```diff
@@ -497,17 +497,19 @@ aux utilisateurs. Pour créer votre ``UserMailer``, créez un fichier
     {
         public function welcome($user)
         {
-            $this->_email->profile('default');
-            $this->_email->to($user->email);
-            $this->_email->subject(sprintf('Welcome %s', $user->name));
+            $this
+                ->to($user->email)
+                ->subject(sprintf('Welcome %s', $user->name))
+                ->template('welcome_mail') // By default template with same name as method name is used.
```
Comment (HavokInspiration) [inline fr/core-libraries/email.rst]: Comment in English

**Comment by HavokInspiration** [comment]
Besides my comments, it looks good to me.

File: fr/tutorials-and-examples/blog/part-two.rst [hunk 4]
```diff
@@ -632,14 +632,18 @@ articles, ainsi :
     </table>
 
 Utiliser :php:meth:`~Cake\\View\\Helper\\FormHelper::postLink()` permet de
-créer un lien qui utilise du Javascript pour supprimer notre post en faisant
-une requête POST. Autoriser la suppression par une requête GET est dangereux à
-cause des robots d'indexation qui peuvent tous les supprimer.
+créer un lien qui utilise du JavaScript pour supprimer notre post en faisant
```
Comment (antograssiot) [inline fr/tutorials-and-examples/blog/part-two.rst]: post -> article

**Comment by cake17** [comment]
@HavokInspiration @antograssiot changes done

**Comment by antograssiot** [comment]
nice @cake17 :smile:

**Pull request merged**